nautobot/core/templates/inc/footer.html

```diff
@@ -6,29 +6,34 @@
         <div class="container-fluid">
             <div class="row">
                 <div class="col-xs-4">
-                    <p class="text-muted">
-                        {% if settings.BRANDING_FILEPATHS.logo and settings.BRANDING_POWERED_BY_URL %}<a href="{{ settings.BRANDING_POWERED_BY_URL }}"><img src="{% static 'img/nautobot_logo.svg' %}" height="20" /> Powered</a> &middot;{% endif %}
-                        {{ settings.HOSTNAME }} {% if request.user.is_authenticated %} (v{{ settings.VERSION }}) {% endif %}
-                    </p>
+                    {% if request.user.is_authenticated or not "HIDE_RESTRICTED_UI"|settings_or_config %}
+                        <p class="text-muted">
+                            {% if settings.BRANDING_FILEPATHS.logo and settings.BRANDING_POWERED_BY_URL %}<a href="{{ settings.BRANDING_POWERED_BY_URL }}"><img src="{% static 'img/nautobot_logo.svg' %}" height="20" /> Powered</a> &middot;{% endif %}
+                            {{ settings.HOSTNAME }} {% if request.user.is_authenticated %} (v{{ settings.VERSION }}) {% endif %}
```

Comment:
@bryanculver @lampwins I'd be real tempted for us to just flat-out remove the `settings.HOSTNAME` from the footer - its meaningless in a Docker deployment and I'm curious if anyone actually finds it ever useful. Thoughts?

Reply:
Probably better to use the banner header for instance validation but I can some nominal value with the Docker deployment model for multi-instance. Help pin down where your logs might be.